In [6]:
#Import library 
import pandas as pd
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt

#data importing

Sale=pd.read_excel('1 Sales Data.xlsx')

Sale.head(5)

Invoice Number       Date             Time   Zone Manager Customer  \
0        102234.0 2010-01-07  10:25:37.200000   East   Bruce     Polo   
1        102235.0 2010-01-07         12:00:00   East   Jacob      EEE   
2        102235.0 2010-01-07         14:19:12   East  Melvin    Marco   
3        102236.0 2010-01-07         12:57:00  South  Alicia      EEE   
4        102236.0 2010-01-07         14:51:36   East    Paul      Zee   

  Product   price  Units Sold    sales  
0     A001  485.0        42.0  20370.0  
1     B227  292.0        71.0  20732.0  
2     A009  544.0        87.0  47328.0  
3     D440  385.0        51.0  19635.0  
4      WP6  173.0        69.0  11937.0

### Check is there any null values in the data. Yes/ No

In [7]:
Sale.isnull().any()

Invoice Number    False
Date              False
Time              False
Zone              False
Manager           False
Customer          False
Product           False
price             False
Units Sold        False
sales             False
dtype: bool

### Find total sales.

In [8]:
a=Sale['sales'].sum()
a

1085806200.0

### Find total number of products sold.

In [9]:
b=Sale['Product '].value_counts().sum()
b

26083

### Find maximum Sales

In [10]:
c=Sale['sales'].max()
c

582000.0

### Find Minimum Sales

In [11]:
c=Sale['sales'].min()
c

865.0

### Find the Average Sales

In [12]:
d=Sale['sales'].mean()
d

41628.88471418165

### Find the Range of Sales

In [14]:
f=865.0-582000.0
f

-581135.0

### Find the Standard Deviation of Sales

In [18]:
e=Sale['sales'].std()
e

30413.734244059717

### Find Average Sales by each zone

In [19]:
g=Sale.groupby('Zone').mean()['sales']
g

Zone
East     40856.468783
North    41933.083101
South    40869.279839
West     42309.618582
Name: sales, dtype: float64

### Find maximum sales by each zone.

In [20]:
g=Sale.groupby('Zone').max()['sales']
g

Zone
East     162656.0
North    165300.0
South    157760.0
West     582000.0
Name: sales, dtype: float64

### Find minimum sales by each zone.

In [21]:
g=Sale.groupby('Zone').min()['sales']
g

Zone
East     1015.0
North    1038.0
South     865.0
West      865.0
Name: sales, dtype: float64

### Give an Inference from the data

# Conclusion


Check is there any null values in the data. Yes/ No

Find total sales.- 1085806200.0

Find total number of products sold. - 26083

Find maximum Sales - 582000.0

Find Minimum Sales - 865.0

Find the Average Sales - 41628.88471418165

Find the Range of Sales - -581135.0

Find the Standard Deviation of Sales - 30413.734244059717

Find Average Sales by each zone 

 Zone

 East     40856.468783

 North    41933.083101

 South    40869.279839

 West     42309.618582

 Name: sales, dtype: float64

Find maximum sales by each zone.

 Zone

 East     162656.0

 North    165300.0

 South    157760.0

 West     582000.0

 Name: sales, dtype: float64


Find minimum sales by each zone.

Zone

East     1015.0

North    1038.0

South     865.0

West      865.0

Name: sales, dtype: float64
